### PR TITLE
Hot wallet docs updates

### PR DESCRIPTION
**Pull request by britttttk**
Part of #837

### CODE REVIEW

File: docs/CreateWallet.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Besides using an [existing wallet](./WalletSetup.md#use-an-existing-wallet) you can create a brand-new **wallet within your BTCPay Server**.
 
 :::warning
-If you are using a [third-party host](./ThirdPartyHosting.md), this option needs to be explicitly enabled by the server admin. Generating a new wallet in an environment you are not sure is trustworthy, is discouraged.
+If you are using a [third-party host](./ThirdPartyHosting.md), this option needs to be explicitly enabled by the server admin. **Do not enable** a hot wallets, unless you've read the [Hot wallet documentation](./HotWallet.md) and have the use-case for this advanced feature.
```
Comment (Zaxounette) [inline docs/CreateWallet.md:4]:
```suggestion
If you are using a [third-party host](./ThirdPartyHosting.md), this option needs to be explicitly enabled by the server admin. **Do not enable** hot wallets, unless you've read the [Hot wallet documentation](./HotWallet.md) and have the use-case for this advanced feature.
```

**Review by Zaxounette — APPROVED** [review]
Typoe.

Other than that, LGTM :)

File: docs/CreateWallet.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Besides using an [existing wallet](./WalletSetup.md#use-an-existing-wallet) you can create a brand-new **wallet within your BTCPay Server**.
 
 :::warning
-If you are using a [third-party host](./ThirdPartyHosting.md), this option needs to be explicitly enabled by the server admin. Generating a new wallet in an environment you are not sure is trustworthy, is discouraged.
+If you are using a [third-party host](./ThirdPartyHosting.md), this option needs to be explicitly enabled by the server admin. **Do not enable** hot wallets, unless you've read the [Hot wallet documentation](./HotWallet.md) and have the use-case for this advanced feature.
 :::
 
 ## How to create a new wallet
```
Comment (dennisreimann) [inline docs/CreateWallet.md:7]:
```suggestion
```

File: docs/CreateWallet.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Besides using an [existing wallet](./WalletSetup.md#use-an-existing-wallet) you can create a brand-new **wallet within your BTCPay Server**.
```
Comment (dennisreimann) [inline docs/CreateWallet.md:1]:
```suggestion
# How to create a new wallet

Besides using an [existing wallet](./WalletSetup.md#use-an-existing-wallet) you can create a brand-new **wallet within your BTCPay Server**.
```

**Review by Zaxounette — APPROVED** [review]

**Review by dennisreimann — APPROVED** [review]

**Comment by dennisreimann** [comment]
If the linkcheck continues to fail I'll do a single pass fix once we merged all of the wallet related PRs …

**Pull request merged**